components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -1450,10 +1456,32 @@ - (UIEdgeInsets)textInsets:(UIEdgeInsets)defaultInsets {
   // .bottom = underlineOffset + MDCTextInputControllerBaseDefaultPadding
   // Legacy default has an additional padding here but this version does not.
   textInsets.bottom = underlineOffset + MDCTextInputControllerBaseDefaultPadding;
-
   return textInsets;
 }
 
+// This method is used to calculate the actual number of lines for the label at the time.
```

Comment:
Consider deleting this first line because it doesn't add any new information to the method signature.